addons/uh60_mfd/config/MFD/pages/monospace.hpp

```diff
@@ -0,0 +1,53 @@
+borderLeft=0;
+borderRight=0;
+borderTop=0;
+borderBottom=0;
+color[]=common_white;
+enableParallax=0;
+font="EtelkaMonospacePro";
```

Comment:
I assume this font is messing up the jvmf spacing? https://cdn.discordapp.com/attachments/811711710666555422/920686026488430642/unknown.png

Reply:
fixed